pkg/asset/installconfig/azure/mock/azureclient_generated.go

```diff
@@ -38,21 +38,6 @@ func (m *MockAPI) EXPECT() *MockAPIMockRecorder {
 	return m.recorder
 }
 
-// GetVirtualNetwork mocks base method.
```

Comment:
Were these supposed to change, or was this the result of a prior `./hack/go-genmock.sh`?
- `pkg/asset/installconfig/azure/mock/azureclient_generated.go`
- `pkg/asset/installconfig/gcp/mock/gcpclient_generated.go`
- `pkg/asset/installconfig/kubevirt/mock/client_generated.go`

Reply:
They wherre not supposed to change due to my changes but for some reason they did.
I guess someone forgot to generate them after updating instalconfig.go

Reply:
Are they required to change for your PR to merge? I'd prefer to pull them out of this PR if not.

Reply:
I agree. Let's leave them out. We'll put up a separate PR with these changes.